website/app/components/doc/icons-list/index.hbs

```diff
@@ -10,7 +10,9 @@
     @filterQuery={{@searchQuery}}
     @onInput={{@searchIcons}}
     data-test="icons-filter"
+    aria-describedby="icons-filter-description"
   />
+  <span id="icons-filter-description" class="sr-only">Note: Icon list will automatically update as you type.</span>
```

Comment:
Another viable option would be to place this inside of the `Doc::Form::Filter` component itself; the outcome will be the same either way.